java/gradle.htmlui/src/org/netbeans/modules/gradle/htmlui/resources/root_build.gradle.fmk

```diff
@@ -18,15 +18,14 @@
     under the License.
 
 -->
-<#noparse>
-plugins {
-    id "me.tatarka.retrolambda" version "3.7.1"
```

Comment:
Just out of curiosity, is retrolambda no longer required?

Reply:
Right, [retrolambda is no longer required](http://wiki.apidesign.org/wiki/Bck2Brwsr#Version_0.50).